cognite_toolkit/_cdf_tk/templates/_templates.py

```diff
@@ -50,7 +50,14 @@ def build_config(
 
     config.validate_environment()
 
-    available_modules = {module.name for module, _ in iterate_modules(source_dir)}
+    available_modules_raw = Counter(module.name for module, _ in iterate_modules(source_dir))
+    if duplicate_modules := {module for module, count in available_modules_raw.items() if count > 1}:
+        print(f"  [bold red]ERROR:[/] Found the following duplicated module names in {source_dir.name}:")
+        for module in duplicate_modules:
+            print(f"    {module}")
```

Comment:
won't this just print 

"module a"
"module a"

Or will it contain the full path to keep them distinct?